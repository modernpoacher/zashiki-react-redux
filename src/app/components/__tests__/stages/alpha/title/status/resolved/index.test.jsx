import React from 'react'
import renderer from 'react-test-renderer'

import Resolved from '#app/components/stages/alpha/title/status/resolved'

describe('#app/components/stages/alpha/status/title/resolved', () => {
  describe('Always', () => {
    it('renders', () => {
      const component = (
        <Resolved
          title='MOCK TITLE'
        />
      )

      expect(renderer.create(component).toJSON())
        .toMatchSnapshot()
    })
  })
})
