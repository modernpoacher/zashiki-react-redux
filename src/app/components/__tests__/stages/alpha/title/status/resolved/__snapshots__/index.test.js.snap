// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/alpha/status/title/resolved renders 1`] = `ShallowWrapper {}`;
